.github/workflows/build.yml

```diff
@@ -339,3 +342,38 @@ jobs:
       run: make pythoninfo
     - name: Tests
       run: xvfb-run make buildbottest TESTOPTS="-j4 -uall,-cpu"
+
+  check:  # This job does nothing and is only used for the branch protection
```

Comment:
"All required checks pass"

Reply:
Job ID: `s/check/all-required-green/`

Reply:
Patched. Marking the conversation as resolved.